Configuration/StandardSequences/python/SimL1EmulatorRepack_FullMC_cff.py

```diff
@@ -60,6 +67,8 @@ def _print(ignored):
 # CSC TPs
 simCscTriggerPrimitiveDigis.CSCComparatorDigiProducer = 'unpackCSC:MuonCSCComparatorDigi'
 simCscTriggerPrimitiveDigis.CSCWireDigiProducer       = 'unpackCSC:MuonCSCWireDigi'
+# GEM
+run3_GEM.toModify(simMuonGEMPadDigis, InputCollection = 'unpackGEM')
```

Comment:
This is the same as what currently in SimL1EmulatorRepack_Full_cff.py, therefore I imagine it may work and my question is probably silly...

But what happens if `run3_GEM` and not `stage2L1Trigger`? 'unpackGEM' is feed as InputCollection to `simMuonGEMPadDigis` [[1]](https://github.com/cms-sw/cmssw/pull/40461/files#diff-6afbf763bc54eac6d2fb579ea378604e8337d4492446134e43419aae84046af6R71), but the `SimL1EmulatorTask` is not updates with the `unpackGEM` module in it [[2]](https://github.com/cms-sw/cmssw/pull/40461/files#diff-6afbf763bc54eac6d2fb579ea378604e8337d4492446134e43419aae84046af6R129)

Reply:
Well, I can only guess as a non-L1T expert.

 - This file only really works for `stage2L1Trigger`, according to the comments up top ("only supports Stage 2 eras for now."), modulo the fact that the comments are mistakenly swapped (the first comment, "use a legacy version", should show for `~stage2L1Trigger`, not for `stage2L1Trigger`)
 - In practise, I think `stage2L1Trigger` applies to nearly anything after 2015, so "`run3_GEM` and not `stage2L1Trigger`" should never happen (in principle..).
 
 Let me fix the comments, since I would like to backport this..

Reply:
Thank you Marino, this is also my understanding.
However, even if so, the clean way to implement I imagine it should be
```suggestion
(stage2L1Trigger & run3_GEM).toModify(simMuonGEMPadDigis, InputCollection = 'unpackGEM')
```

Reply:
Following what I see elsewhere, e.g.
https://github.com/cms-sw/cmssw/blob/a45ef0567719756ab6b998f0bd1611151ca695f9/L1Trigger/L1TMuon/python/simDigis_cff.py#L119
I can change `run3_GEM` to `(stage2L1Trigger & run3_GEM)` also above (and in "Full").

Reply:
>  However, even if so, the clean way to implement I imagine it should be

I cross-posted. Yes, will do (soon).